spec/models/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'David', photo: 'http://', bio: 'Teacher from Mexico') }
+
+  before { subject.save }
+
+  it 'title should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+  it 'bio should be present' do
+    subject.bio = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Kindly add tests for `posts_counter` validation.

- [ ] Include tests for the `top_three` method as contained in your User model